nano/node/lmdb/lmdb.cpp

```diff
@@ -780,6 +786,14 @@ void nano::mdb_store::upgrade_v20_to_v21 (nano::write_transaction const & transa
 	logger.always_log ("Finished creating new final_vote table");
 }
 
+void nano::mdb_store::upgrade_v21_to_v22 (nano::write_transaction const & transaction_a)
```

Comment:
> Should be applicable only if the table is enabled.

It is just the empty table creation for LMDB, which should be finished rather fast. I don't really know where I could move the table creation otherwise if I may not use the version counter as well. How do I check if the table already exists? 😅

Reply:
Btw. thanks for the fast feedback! 😄

Reply:
Np, there is an alternative you can use, `mdb_dbi_open` returns an error when the handle doesn't exist.
Look at this: https://github.com/nanocurrency/nano-node/blob/a37a1480d8a13821dc6b2345ecc15517d64b0e35/nano/node/lmdb/lmdb.cpp#L83

Reply:
> Np, there is an alternative you can use, `mdb_dbi_open` returns an error when the handle doesn't exist. Look at this:
> 
> https://github.com/nanocurrency/nano-node/blob/a37a1480d8a13821dc6b2345ecc15517d64b0e35/nano/node/lmdb/lmdb.cpp#L83

@thsfs I removed the upgrade method completely and reset the version number back to 21. Is it fine if the reverse_links table is created in `nano::mdb_store::open_databases` with the other tables, even if the reverse link option is disabled? We don't have direct node config access there, so I would have to either add an extra parameter for this option or add it to `nano::lmdb_config` (which would probably be the solution to go if the option should decide if the table should be created or not).